contracts/proxy/transparent/TransparentUpgradeableProxy.sol

```diff
@@ -15,12 +15,7 @@ import {ProxyAdmin} from "./ProxyAdmin.sol";
  * include them in the ABI so this interface must be used to interact with it.
  */
 interface ITransparentUpgradeableProxy is IERC1967 {
-    /**
-     * @dev Upgrade the implementation of the proxy to `newImplementation`, and subsequently execute the function call
-     * encoded in `data`.
-     *
-     * See {UUPSUpgradeable-upgradeToAndCall}
-     */
+    // See {UUPSUpgradeable-upgradeToAndCall}
```

Comment:
The audit issue referred to missing NatSpec. I think we should do triple dashes and `@dev` to fix the issue correctly (otherwise is not NatSpec). This way the doc will show briefly in code editors (e.g. VSCode) when doing  `ITransparentUpgradeableProxy.upgradeToAndCall(...`

```suggestion
    /// @dev See {UUPSUpgradeable-upgradeToAndCall}
```